iris-mpc-upgrade/build.rs

```diff
@@ -1,6 +1,6 @@
 fn main() {
     println!("cargo:rerun-if-changed=build.rs");
-    println!("cargo:rerun-if-changed=protos/reshare.proto");
+    println!("cargo:rerun-if-changed=proto/reshare.proto");
```

Comment:
the original here should be correct, but actually tonic_build already prints this internally so this line is not needed at all

Reply:
thanks, will remove it